### ISSUE TITLE
docker build failed: Error: Errno is not a subclass of Exception

### ISSUE THREAD

**Issue by VSilantyev** [cloudamqp/amqproxy#31]
```
docker build -t amqproxy:v0.4.4 https://github.com/cloudamqp/amqproxy.git#v0.4.4
Sending build context to Docker daemon  185.3kB
Step 1/8 : FROM crystallang/crystal
latest: Pulling from crystallang/crystal
5bed26d33875: Pull complete 
f11b29a9c730: Pull complete 
930bda195c84: Pull complete 
78bf9a5ad49e: Pull complete 
e27d61588c18: Pull complete 
Digest: sha256:b3f86e938a3de7454d466736c013cf25ed37e378f64bb0583555627b6d2ea90f
Status: Downloaded newer image for crystallang/crystal:latest
 ---> 89062caa0bb5
Step 2/8 : WORKDIR /app
 ---> Running in b3153b37c2be
Removing intermediate container b3153b37c2be
 ---> f6afa181fbf2
Step 3/8 : COPY . .
 ---> 4404ad8c69d8
Step 4/8 : RUN shards build --release --production
 ---> Running in ba4ee679d3f9
Resolving dependencies
Fetching https://github.com/cloudamqp/amq-protocol.cr.git
Installing amq-protocol (0.3.4)
Building: amqproxy
Error target amqproxy failed to compile:
Showing last frame. Use --error-trace for full trace.

In src/amqproxy/upstream.cr:118:17

 118 | rescue ex : Errno | IO::Error | OpenSSL::SSL::Error
                   ^----
Error: Errno is not a subclass of Exception

The command '/bin/sh -c shards build --release --production' returned a non-zero code: 1
```
I have tried multiple releases, master - same error.

**Comment by carlhoerberg** [comment]
ah, yes, a stdlib change in crystal 0.34, the Errno exception is removed. Will fix

**Comment by carlhoerberg** [comment]
fixed in 0.4.5, thank you for reporting
